index.html

```diff
@@ -12,85 +13,187 @@
 
 <body>
 
-	<h1>You got this!  Good luck.</h1>
-
-<!-- Menu Content 
-S&J
-
+<header>
```

Comment:
As there is no `max-width` your site is going till the edge of the display regardless of the display size, if you have wrapped everything in your body in a container it would have been easy to do that.